src/components/Prompt/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Prompt({ isShowed, children, deleteAction, toggleModal }) {
+  return (
+    isShowed && (
+      <div className="prompt" aria-modal="true" role="dialog" id="dialog1">
```

Comment:
Id's must be unique. If you open multiple "delete" buttons. All the "prompts" id's will be the same "dialog1"

Reply:
Thanks for your comments!
Well, I changed our `Prompt` component to `windows.confirm` 😄 .